### PR TITLE
Added SOE specific configs on top of stanford_profile

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- SOE configs and things on top of stanford_profile
  - The 8.x.2.x branch in the repo I started fresh with the very first commit of this repo's history.
  - I pulled in the history from stanford_profile `8.x-2.x` branch. 
  - I renamed files/namespaces etc. 
  - Did a git checkout of the config directory and fixed anything weird.
  - This should put the soe_profile on a "forked" path from stanford_profile.

# Need Review By (Date)
- 2/4

# Urgency
- high

# Steps to Test
1. use branch in https://github.com/SU-SWS/acsf-lelandd8/pull/61
2. observe the soe site still looks the same with a new install or an existing site updated.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,6 +74,24 @@ d9_codeception: &d9_codeception
...
+        command: |
+          git config --global user.email "sws-developers@lists.stanford.edu"
+          git config --global user.name "CircleCI"
+          git pull https://github.com/SU-SWS/stanford_profile.git 8.x-2.x -X ours --no-edit
+          if [[ $(git diff origin/${CIRCLE_BRANCH} --name-only) ]]; then
+            DATE=`date +%Y-%m-%d`
+            git checkout -b ${DATE}
+            git push origin ${DATE}
+            hub pull-request --base=${CIRCLE_BRANCH} --message="Updates from stanford_profile" --assign=pookmish,imonroe
+          fi
```
Comment (imonroe) [inline .circleci/config.yml:93]: This is classy right here.

File: composer.json [hunk 2]
```diff
@@ -178,38 +115,6 @@
...
-                "https://www.drupal.org/project/menu_link_weight/issues/3099139": "https://www.drupal.org/files/issues/2019-12-05/menu_link_weight-target_blank-3099139.patch"
-            },
-            "drupal/paragraphs": {
-                "https://www.drupal.org/project/paragraphs/issues/2901390": "https://www.drupal.org/files/issues/2020-06-25/paragraphs-2901390-51.patch"
-            },
-            "drupal/simplesamlphp_auth": {
-                "https://www.drupal.org/project/simplesamlphp_auth/issues/2936889": "https://www.drupal.org/files/issues/2020-07-31/simplesamlphp_auth-avoid_unexpected_redirect-2936889-12-d8.patch"
-            }
-        }
+        "patches": {}
```
Comment (imonroe) [inline composer.json:118]: Are we relying on stanford_profile applying any relevant patches?

File: config/sync/core.entity_form_display.node.stanford_page.default.yml [hunk 3]
```diff
@@ -84,7 +84,9 @@ content:
         stanford_wysiwyg: 3
         stanford_media_caption: 3
         stanford_spacer: 1
-        stanford_gallery: 12
```
Comment (imonroe) [inline config/sync/core.entity_form_display.node.stanford_page.default.yml:87]: Are we waiting to add gallery paragraphs to soe?

File: config/sync/field.storage.node.layout_builder__layout.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-uuid: 833a0790-a592-43cb-90d6-f15509b9c74a
+uuid: ed8c3733-5cf1-4e6d-940e-350e7c8d76b8
```
Comment (imonroe) [inline config/sync/field.storage.node.layout_builder__layout.yml:1]: I'm not sure how much difference it makes, but if we make the uuids match wherever we can, it cuts down on the diffs between profiles which simplifies reconciling changes.  I'll defer to Mike on this sort of thing though.

File: config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml [hunk 5]
```diff
@@ -40,18 +40,27 @@ settings:
...
+        weight: 8
         enabled: false
       stanford_person_cta:
         enabled: true
-        weight: -15
+        weight: 11
       stanford_schedule:
         enabled: true
-        weight: -13
+        weight: 12
```
Comment (imonroe) [inline config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml:50]: This is another thing where I feel like maybe we should just make them match for simplicity, unless there's a strong reason not to.

**Review by imonroe — COMMENTED** [review]
Looks pretty good; I'm glad it's reconciling some of the more annoying settings like weights, etc.  I want to install it locally to test before I give a thumbs up.

File: config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml (hunk 5 above)
Comment (imonroe) [reply config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml:50]: I guess that's precisely what's happening here.  Thumbs up on that.

File: composer.json (hunk 2 above)
Comment (pookmish) [reply composer.json:118]: Yeah. they get inherited. Much less to worry about here.

File: config/sync/field.storage.node.layout_builder__layout.yml (hunk 4 above)
Comment (pookmish) [reply config/sync/field.storage.node.layout_builder__layout.yml:1]: The uuid in this pr matches that of the production sites. At some point the UUID on soe_profile `8.x-1.x` changed for whatever reason. We have to keep UUIDs the the same especially for field storage. During configuration import, if the UUID is different, Drupal will delete the field storage and the database table and then recreate it. Doing that loose any client data.

**Comment by pookmish** [comment]
Assuming this is approved due to the approval and merger of the stack https://github.com/SU-SWS/acsf-lelandd8/pull/61

**Pull request merged**